docs/get-started/deployment-and-usage.md

```diff
@@ -162,6 +162,22 @@ nfd-master args, in which case nfd-master verifies that the NodeName presented
 by nfd-worker matches the Common Name (CN) of its certificate. This means that
 each nfd-worker requires a individual node-specific TLS certificate.
 
+### DRAFT: cert-manager
+
+cert-manager can be used to manage TLS between nfd-master and the nfd-worker
+pods.
+
+```bash
+# Steps
+1. Deploy cert-manager
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
+2. generate tls.key and tls.crt
```

Comment:
Should we use names `ca.key` and `ca.crt` here to make it clear what we're doing here

Reply:
`tls.[key|crt]` are mandatory fields with the CA Issuer: "Take note of the index keys used for each field as these are required in order for cert-manager to find the certificate and key. "